appServer/index.js

```diff
@@ -0,0 +1,32 @@
+/*
+    Server entry file.
+    No changes are required here to load a plugin
+*/
+
+'use strict';
+
+const express = require('express');
+const app = express();
+const glob = require('glob');
+
+/*
+    Loading all the plugins from plugin directory
+*/
+
+glob("./plugins/**/*.routes.js", function (er, files) {
```

Comment:
Make the plugin folder as a variable, define it at the top and append /**/*.routes.js